sdk/identity/Azure.Identity/src/Constants.cs

```diff
@@ -20,6 +20,8 @@ internal class Constants
 
         public static readonly TimeSpan SharedTokenCacheAccessRetryDelay = TimeSpan.FromMilliseconds(600);
 
+        public const string DefaultRedirectUrl = "http://localhost";
```

Comment:
Do we need a default port here?

Reply:
This is the current default value that's being used. I just moved it into a constant.